### PR TITLE
Switch project to use Python 3

### PR DESCRIPTION
**Pull request by andrewda**

### CODE REVIEW

**Comment by andrewda** [comment]
Does this break backwards compatibility with Python 2.7? If not, I'm good to merge.

File: predict.py [hunk 1]
```diff
@@ -54,8 +54,9 @@ def _main_(args):
     ###############################
     #   Load trained weights
     ###############################
-
-    print weights_path
+    if weights_path is None:
+        weights_path = config["train"]["saved_weights_name"]
+    print(weights_path)
```
Comment (andrewda) [inline predict.py]: this should be done in another PR

Comment (andrewda) [reply predict.py]: also we're using single quotes

**Comment by andrewda** [comment]
Looks like this is fully backwards compatible. @JacksonCoder - please take a look at comments above.

**Comment by JacksonCoder** [comment]
I redid the commit to not include the part you specified. Is it good to merge?
